mirdata/datasets/haydn_op20.py

```diff
@@ -0,0 +1,309 @@
+        music21.stream.Score: score in music21 format
+        List[Tuple[Any, Any]]: roman numerals
+    """
+    score = music21.converter.parse(fhandle.name, format="humdrum")
+    rna = {rn.offset: rn for rn in list(score.flat.getElementsByClass("RomanNumeral"))}
+    score.remove(rna, recurse=True)
+    rna_clean = [(offset, rn) for offset, rn in rna.items() if rn]
+    return score, rna_clean
+
+
+@io.coerce_to_string_io
+def load_score(fhandle: TextIO):
+    """Load haydn op20 score with annotations from a file with music21 format (music21.stream.Score).
+
+    Args:
+        fhandle (str or file-like): path to score
+
+    Returns:
+        music21.stream.Score: score in music21 format
+
+    """
+    score, rna = split_score_annotations(fhandle)
+    return score
+
+
+@io.coerce_to_string_io
+def load_key(fhandle: TextIO, resolution=28):
+    """Load haydn op20 key data from a file
+
+    Args:
+        fhandle (str or file-like): path to key annotations
+
+    Returns:
+        List[dict]: musical key data and time
+
+    """
+    _, rna = split_score_annotations(fhandle)
+    annotations = []
+    for offset, rn in rna:
+        time = int(round(float(offset * resolution)))
+        tonicizedKey = rn.secondaryRomanNumeralKey
+        key = tonicizedKey or rn.key
+        annotations.append({"time": time, "key": key})
+    return annotations
+
+
+@io.coerce_to_string_io
+def load_midi_path(fhandle: TextIO):
+    """Load path to midi file of haydn op20 musical piece
+
+    Args:
+        fhandle (str or file-like): path to score file
+
+    Returns:
+        str: midi file path
+
+    """
+    midi_path = os.path.splitext(fhandle.name)[0] + ".midi"
+    if not os.path.exists(midi_path):
+        score, _ = split_score_annotations(fhandle)
+        score.write("midi", fp=midi_path)
+    return midi_path
+
+
+@io.coerce_to_string_io
+def load_roman_numerals(fhandle: TextIO, resolution=28):
+    """Load haydn op20 roman numerals data from a file
+
+    Args:
+        fhandle (str or file-like): path to roman numeral annotations
+
+    Returns:
+        List[dict]: musical roman numerals data and time
+
+    """
+    _, rna = split_score_annotations(fhandle)
+    annotations = []
+    for offset, rn in rna:
+        time = int(round(float(offset * resolution)))
+        figure = rn.figure
+        annotations.append({"time": time, "roman_numeral": figure})
+    return annotations
+
+
+@io.coerce_to_string_io
+def load_chords(fhandle: TextIO, resolution: int = 28):
+    """Load haydn op20 chords data from a file
+
+    `Args:
+        fhandle (str or file-like): path to chord annotations
+
+    Returns:
+        List[dict`]: musical chords data and time
+
+    """
+    _, rna = split_score_annotations(fhandle)
+    annotations = []
+    for offset, rn in rna:
+        time = int(round(float(offset * resolution)))
+        chord = rn.pitchedCommonName
+        annotations.append({"time": time, "chord": chord})
+    return annotations
```

Comment:
Why not `ChordData`? This dataset is starting to feel a bit divergent to `mirdata` conventions. Let's make sure it makes sense for it to be (let's discuss it offline on Monday)

Reply:
Here it is the same. Music21 is omnipresent in the symbolic/scores community that is why I am including key and chord with this format. Moreover, the format of the chord is music21.chord.Chord because it is very easy to parse it to another notation or to pitch representation or things like that.

Reply:
+1 to @magdalenafuentes 's comment.

Reply:
@rabitt but the format it is not the "beeatles format" supported by jams. However the music21.chord data type supports some different formats

Reply:
@rabitt what I do?